frontend/src/concepts/modelRegistry/types.ts

```diff
@@ -85,6 +85,8 @@ export type ModelRegistryBase = {
   id: string;
   name: string;
   externalID?: string;
+  author?: string;
+  owner?: string;
```

Comment:
We shouldn't add either of these to `ModelRegistryBase` - I know I had originally added author here when we were pairing on this @YuliaKrimerman but I was wrong, sorry. I hadn't realized it was called `owner` for RegisteredModels.

The type updates we need are:

ModelRegistryBase - no change
RegisteredModel - add `owner`
ModelArtifact - no change (don't remove `author`)
ModelVersion - no change (don't remove `author`)